ena-submission/ruff.toml

```diff
@@ -0,0 +1,75 @@
+# Exclude a variety of commonly ignored directories.
```

Comment:
I'm a bit worried about maintainability - we have three other ruff files, one is in the main directory - I think this will format all subdirectories. Do we want to remove the other ruff files and just have one ruff file in the main directory so that we use the same formatting standards everywhere? Ruff should be able to find the file in the root even when run from subdirectories